src/libsyntax/feature_gate.rs

```diff
@@ -1622,12 +1622,6 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
                 }
             }
 
-            ast::ItemKind::TraitAlias(..) => {
-                gate_feature_post!(&self, trait_alias,
```

Comment:
I believe we still want this -- this is what requires us to use a feature-gate, no? You can change the text to "trait aliases are still experimental" or something though.

Reply:
Ah yes, fair enough.